### PR TITLE
Page bicycles

### PR DESCRIPTION
**Pull request by NazariiSova**

### CODE REVIEW

File: src/components/Bicicletas-catalog/BicicletasCatalog.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <img className="bike-img" src={img} alt="bike" />
+        <div className="bike-price">{price}</div>
+      </div>
+      <div className="bike-title-and-dec">
+        <div className="bike-title">{title}</div>
+        <img src={dec} alt="dec" className="bike-dec" />
+      </div>
+      <div className="bike-description">
+        A Magic Migth é a melhor Bikcraft já criada pela nossa equipe. Ela vem
+        equipada com os melhores acessórios, o que garante maior velocidade.{" "}
```
Comment (taras-zherebetskyi) [inline src/components/Bicicletas-catalog/BicicletasCatalog.jsx]: Change description for different components

File: src/components/Bicicletas-catalog/BicicletasCatalog.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import material from "../img/Icons/material.png";
+import speed from "../img/Icons/speed.png";
+import tracker from "../img/Icons/tracker.png";
+
+import dec from "../img/dec.svg";
+
+export const BicicletasCatalog = ({ isDarkBackground, img, price, title }) => {
+  let Background = isDarkBackground ? "-dark" : "-light";
+  return (
+    <div className={`bike-container${Background}`}>
```
Comment (taras-zherebetskyi) [inline src/components/Bicicletas-catalog/BicicletasCatalog.jsx]:
```suggestion
  let Background = isDarkBackground ? "dark" : "light";
  return (
    <div className={`bike-container-${Background}`}>
```

Comment (taras-zherebetskyi) [inline src/components/Bicicletas-catalog/BicicletasCatalog.jsx]:
```suggestion
  let Background = isDarkBackground ? "-dark" : "-light";
  return (
    <div className={`bike-container bike-container--${Background}`}>
```

File: src/pages/Bicicletas/Bicicletas.page.jsx [hunk 3]
```diff
@@ -1,25 +1,30 @@
...
+  return (
+    <div className="bicicletas-page-container">
+      <div className="header-container">
+        <div className="header-small-text">ESCOLHA A MELHORA PARA VOCÊ</div>
+        <div className="header-title">nossas bicicletas</div>
+      </div>
+      <div className="bicicletas-catalog-container">
+        <BicicletasCatalog isDarkBackground={false} img={bicicleta1} price={"R$ 4.999"} title={"Nimbus Stark"}/>
+        <BicicletasCatalog isDarkBackground={true} img={bicicleta2} price={"R$ 2.499"} title={"Magic Migth"}/>
+        <BicicletasCatalog isDarkBackground={false} img={bicicleta3} price={"R$ 3.999"} title={"Mebula Cosmic"}/>
```
Comment (taras-zherebetskyi) [inline src/pages/Bicicletas/Bicicletas.page.jsx]: Make array and map it

File: src/pages/Bicicletas/bicicletasPage.scss [hunk 4]
```diff
@@ -33,3 +33,10 @@
 
   }
 }
+
+.bicicletas-catalog-container{
+display: flex;
+flex-direction: column;
+align-items: end;
```
Comment (taras-zherebetskyi) [inline src/pages/Bicicletas/bicicletasPage.scss]:
```suggestion
align-items: flex-end;
```

**Review by taras-zherebetskyi — CHANGES_REQUESTED** [review]
Very nice 🔥

File: src/components/Bicicletas-catalog/BicicletasCatalog.jsx (hunk 2 above)
Comment (NazariiSova) [reply src/components/Bicicletas-catalog/BicicletasCatalog.jsx]: done

Comment (NazariiSova) [reply src/components/Bicicletas-catalog/BicicletasCatalog.jsx]: done

File: src/pages/Bicicletas/bicicletasPage.scss (hunk 4 above)
Comment (NazariiSova) [reply src/pages/Bicicletas/bicicletasPage.scss]: done

File: src/components/Bicicletas-catalog/BicicletasCatalog.jsx (hunk 1 above)
Comment (NazariiSova) [reply src/components/Bicicletas-catalog/BicicletasCatalog.jsx]: done

File: src/pages/Bicicletas/Bicicletas.page.jsx (hunk 3 above)
Comment (NazariiSova) [reply src/pages/Bicicletas/Bicicletas.page.jsx]: done

File: src/components/Bicicletas-catalog/BicicletasCatalog.jsx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+      <div className="bike-img-container">
+        <img className="bike-img" src={img} alt="bike" />
+        <div className="bike-price">{price}</div>
+      </div>
+      <div className="bike-title-and-dec">
+        <div className="bike-title">{title}</div>
+        <img src={dec} alt="dec" className="bike-dec" />
+      </div>
+      <div className="bike-description">
+        {`A ${title}é a melhor Bikcraft já criada pela nossa equipe. Ela vem
```
Comment (taras-zherebetskyi) [inline src/components/Bicicletas-catalog/BicicletasCatalog.jsx:24]:
```suggestion
        {`A ${title} é a melhor Bikcraft já criada pela nossa equipe. Ela vem
```

U need space here

File: src/pages/Home/Home.page.jsx [hunk 6]
```diff
@@ -39,7 +39,7 @@ const HomePage = () => {
             Bicicletas elétricas de alta precisão e qualidade, feitas sob medida
             para o cliente. Explore o mundo na sua velocidade com a Bikcraft.
           </div>
-          <button className="header-post-button">Selecione a sua</button>
+          <button className="header-post-button" > <Link className="header-post-button-link"to="/bicicletas">Selecione a sua</Link></button>
```
Comment (taras-zherebetskyi) [inline src/pages/Home/Home.page.jsx:42]: Use only link here

**Review by taras-zherebetskyi — APPROVED** [review]
So nice👍 
Fix comments & merge

File: src/components/Bicicletas-catalog/BicicletasCatalog.jsx (hunk 5 above)
Comment (NazariiSova) [reply src/components/Bicicletas-catalog/BicicletasCatalog.jsx:24]: done

File: src/pages/Home/Home.page.jsx (hunk 6 above)
Comment (NazariiSova) [reply src/pages/Home/Home.page.jsx:42]: done

**Pull request merged**